### ISSUE TITLE
Uncaught TypeError: Cannot read property 'split' of undefined

### ISSUE THREAD

**Issue by luisjuniorj** [fenichelar/ember-simple-auth-token#139]
I just cant "not to login" how to handle invalid credentials? if i send a null token, the error persists...

**Comment by eluciano11** [comment]
Hi @luisjuniorj , is your backend returning a property named `token` after the authentication? By default this add-on uses the name `token` as the `tokenPropertyName`. If your backend returns something like `auth` you should change your `tokenPropertyName` to `auth` in your `config/environment.js`

Example:

``` javascript
  ENV['ember-simple-auth-token'] = {
    tokenPropertyName: 'auth'
  };
```

**Comment by luisjuniorj** [comment]
Hi, the problem is the authentication failed so there is no token
Em 27/03/2016 2:41 PM, "Emmanuel Luciano" notifications@github.com
escreveu:

**Comment by eluciano11** [comment]
ok, would you like to make a PR to fix this? If you can't we can add it to our queue and we'll get around it later this week.

**Comment by eluciano11** [comment]
Should be fixed in #147
